css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+h2 {
+  font-family: "Brandon Grotesque", sans-serif;
+  color: rgb(236 93 87);
+  letter-spacing: 2.5px;
+  font-weight: bold;
+  text-align: center;
+  font-size: 20px;
+  text-transform: uppercase;
+  margin-bottom: -10px;
+}
+
+form {
+  display: block;
+}
+
+label {
+  display: block;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: bold;
+  color: #2b2b2b;
+  font-size: 14px;
+  text-transform: uppercase;
+  margin: 18px 0 5px 10px;
+}
+
+button {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: 1rem;
+  color: white;
+  background-color: rgb(236 93 87);
+  border: none;
+  border-radius: 20px;
+  padding: 7px 25px;
+  margin: 12px 0 0 400px;
```

Comment:
Please avoid using margin in order to push button to the right — hint: you should be able to do the same using flexbox